test/playback.test.js

```diff
@@ -90,7 +91,7 @@ QUnit.test('replay', function(assert) {
   });
 });
 
-QUnit.skip('playlist with fmp4 and ts segments', function(assert) {
+QUnit.test('playlist with fmp4 segments', function(assert) {
```

Comment:
this test was intended to be for a playlist with mixed ts and fmp4 segments, this change is **not** equivalent

Reply:
agreed, I have an issue open for fixing mixed ts and fmp4 segments. For now it seemed like a test for only fmp4 would be OK.

Reply:
it's theoretically fine but you will need to rewrite this test later when mixed fmp4 and ts is fixed, either works.